cfy_manager/main.py

```diff
@@ -565,9 +574,11 @@ def _all_services_installed():
                for service_name in config[SERVICES_TO_INSTALL])
 
 
-def _all_services_configured():
-    return all(service_name in get_configured_services()
-               for service_name in config[SERVICES_TO_INSTALL])
+def _all_services_configured(required_services=set()):
+    configured_services = set(get_configured_services())
+    services_to_install = set(config[SERVICES_TO_INSTALL])
+    return (configured_services == services_to_install and
+            required_services.issubset(configured_services))
```

Comment:
I don't think issubset is right for this- the DB operations need the node to be database or manager (could be both, but getting them installed like that would be 'fun').